--- tm_def.h	2018-08-29 16:10:38.777483221 +0800
+++ tm_def-new.h	2018-08-29 16:10:19.781266438 +0800
@@ -25,12 +25,14 @@
 
 
 #if (__GLIBC__ == 2 && __GLIBC_MINOR__ > 19)
+/*
 TM_PURE int fprintf (FILE *__restrict __stream, const char *__restrict __fmt, ...);
 TM_PURE int sprintf (char *__restrict __s, const char *__restrict __format, ...) __THROWNL;
 TM_PURE int snprintf (char *__restrict __s, size_t __maxlen, const char *__restrict __format, ...)
 	__THROWNL __attribute__ ((__format__ (__printf__, 3, 4)));
 TM_PURE int vsprintf (char *__restrict __s, const char *__restrict __fmt, _G_va_list __ap);
 TM_PURE int vsnprintf (char *__restrict __s, size_t __n, const char *__restrict __fmt, _G_va_list __ap);
+*/
 #endif
 
 TM_PURE size_t __builtin_object_size (void * ptr, int type);
